### DISCUSSION TITLE
Problem in running pile penetration into saturated soil

### DISCUSSION THREAD

**Discussion by PouyaCR** [Anura3D/Anura3D_OpenSource#139] [Q&A] [answered]
Hello
I am going to run my pile penetration into saturated soil and I use fully-coupled model. But, the problem is that it takes too long to solve the steps even I use Core i9 PC. Do you have an idea? My model is very small with 1 material point  per element.

**Comment by luisez1988** [comment, marked as answer]
Hello, 

There could be many reasons why this is happening. Could you provide more information about the velocity of penetration, how long the penetration takes (i.e., real-time), element size, and the constitutive model you are using?

If the velocity is small, say CPT-like, try mass scaling, accelerating the penetration process.

**Reply by PouyaCR** [reply]
Dear Luisez
I did 2d axisymmetric analyses to model a pile penetration. This is the result: Do you have previous version of Anura? for example, Anura 2021 or 2020??

https://github.com/Anura3D/Anura3D_OpenSource/assets/145875446/88164cb3-7241-4bdc-a830-132688111b00

Regards

**Comment by PouyaCR** [comment]
Dear Dr Luisez
I have attached here my numerical modeling result. It is the 2d
axisymmetric pile penetration problem. Its outcome is strange. Also, I have
attached my GiD file for your consideration.  Mybe, there is an error in my
modeling. Otherwise, the problem is from Anura 3d.

Regards
Pouya Zahedi
PhD Student
Department of Civil, Structural and Environmental Engineering
Trinity College Dublin, The University of Dublin
Dublin
Ireland
Linkedin <https://www.linkedin.com/in/pouya-zahedi-489651242>/ICRAG
<https://www.icrag-centre.org/people/pouyazahedi.html>
